src/cmd/flux-kvs.c

```diff
@@ -304,37 +309,85 @@ int main (int argc, char *argv[])
     return (exitval);
 }
 
-static void output_key_json_object (const char *key, json_t *o)
+static void kv_printf (const char *key, int maxcol, const char *fmt, ...)
+{
+    va_list ap;
+    int rc;
+    char *val, *kv, *p;
+    bool overflow = false;
+
+    assert (maxcol == 0 || maxcol > 3);
```

Comment:
just realized, I don't think there's a check for a reasonable maxcol in callers of this function.  Should a check be added somewhere?  Or perhaps there should just be a generic "can't output with such small width" kind of error message?

Reply:
Yes you're right.